scripts/world_bank/WorldBankIndicators.csv

```diff
@@ -1,8 +1,52 @@
-VC.BTL.DETH,Battle-related deaths (number of people),"Battle-related deaths are deaths in battle-related conflicts between warring parties in the conflict dyad (two conflict units that are parties to a conflict). Typically, battle-related deaths occur in warfare involving the armed forces of the warring parties. This includes traditional battlefield fighting, guerrilla activities, and all kinds of bombardments of military units, cities, and villages, etc. The targets are usually the military itself and its installations or state institutions and state representatives, but there is often substantial collateral damage in the form of civilians being killed in crossfire, in indiscriminate bombings, etc. All deaths--military as well as civilian--incurred in such situations, are counted as battle-related deaths.","Uppsala Conflict Data Program, http://www.pcr.uu.se/research/ucdp/.",dcs:MortalityEvent,measuredValue,count,causeOfDeath,dcs:ICD10/Y35-Y38 ,,,,,,
-TX.VAL.TECH.CD,High-technology exports (current US$),"High-technology exports are products with high R&D intensity, such as aerospace, computers, pharmaceuticals, scientific instruments, and electrical machinery. Data are in current U.S. dollars.","United Nations, Comtrade database through the WITS platform.",dcs:EconomicActivity,measuredValue,amount,economicActivityType,Export,isic,,,,,USDollar
-TX.VAL.SERV.CD.WT,Commercial service exports (current US$),"Commercial service exports are total service exports minus exports of government services not included elsewhere. International transactions in services are defined by the IMF's Balance of Payments Manual (1993) as the economic output of intangible commodities that may be produced, transferred, and consumed at the same time. Definitions may vary among reporting economies.",,dcs:EconomicActivity,measuredValue,amount,economicActivityType,Export,isic,,,,,USDollar
-TX.VAL.MRCH.CD.WT,Merchandise exports (current US$),Merchandise exports show the f.o.b. value of goods provided to the rest of the world valued in current U.S. dollars.,World Trade Organization.,dcs:EconomicActivity,measuredValue,amount,economicActivityType,Export,isic,,,,,USDollar
-TM.VAL.MRCH.WL.CD,Merchandise imports by the reporting economy (current US$),"Merchandise imports by the reporting economy are the total merchandise imports by the reporting economy from the rest of the world, as reported in the IMF's Direction of trade database. Data are in current U.S. dollars.",World Bank staff estimates based data from International Monetary Fund's Direction of Trade database.,dcs:EconomicActivity,measuredValue,amount,economicActivityType,Import,,,,,,USDollar
\ No newline at end of file
+IndicatorCode,ConvertToInt,ExistingStatVar,IndicatorName,SourceNote,Source,populationType,statType,measuredProp,p1,v1,p2,v2,p3,v3,measurementDenominator,scalingFactor,sourceScalingFactor,measurementMethod,unit

```

Comment:
If this will be read by Pandas's read_csv, then it might be nice to add some comments explaining the file's columns (https://stackoverflow.com/q/18366797).  This will help a future maintainer.